.env

```diff
@@ -0,0 +1,2 @@
+PORT=7000
```

Comment:
this file should be in .gitignore

Reply:
It is, but not at same time, apparently. :(